diff -ur libavformat/http.c libavformat/http.c
--- libavformat/http.c	2010-11-18 13:31:16.000000000 +1100
+++ libavformat/http.c	2010-11-08 20:15:32.000000000 +1100
@@ -52,10 +52,11 @@
 } HTTPContext;
 
 #define OFFSET(x) offsetof(HTTPContext, x)
+/* NOTE: MNNLIVE changed default to -1 */
 static const AVOption options[] = {
+{"chunksize", "use chunked transfer-encoding for posts, -1 disables it, 0 enables it", OFFSET(chunksize), FF_OPT_TYPE_INT64, -1, -1, 0 }, /* Default to 0, for chunked POSTs */
-{"chunksize", "use chunked transfer-encoding for posts, -1 disables it, 0 enables it", OFFSET(chunksize), FF_OPT_TYPE_INT64, 0, -1, 0 }, /* Default to 0, for chunked POSTs */
 {NULL}
 };
 static const AVClass httpcontext_class = {
     "HTTP", av_default_item_name, options, LIBAVUTIL_VERSION_INT
 };
